internal/rootcoord/meta_table.go

```diff
@@ -1046,3 +1067,84 @@ func (mt *metaTable) dupMeta() (
 	}
 	return collID2Meta, segID2IndexMeta, indexID2Meta
 }
+
+func (mt *metaTable) AddAlias(collectionAlias string, collectionName string,
+	ts typeutil.Timestamp, ddOpStr func(ts typeutil.Timestamp) (string, error)) error {
+	mt.ddLock.Lock()
+	defer mt.ddLock.Unlock()
+	if _, ok := mt.collAlias2ID[collectionAlias]; ok {
```

Comment:
what if table not exist? did we check any where?

Reply:
This error will be returned. 
```
	id, ok := mt.collName2ID[collectionName]
	if !ok {
		return fmt.Errorf("aliased collection name does not exist, name = %s", collectionName)
	}
```